Formula/grt.rb

```diff
@@ -0,0 +1,29 @@
+class Grt < Formula
+  desc "The Gesture Recognition Toolkit (GRT) for Real-time machine learning."
```

Comment:
These are minor stylistic issues, but please:

- Drop `(GRT)`. The formula name `grt` will be printed in virtually any context where this description will appear.
- Why is “Real-time” capitalized? I don't think it should be.
- Drop the full stop at the end.